avm/res/network/network-manager/routing-configuration/main.bicep

```
@@ -0,0 +1,77 @@
+metadata name = 'Network Manager Routing Configurations'
+metadata description = '''This module deploys an Network Manager Routing Configuration.
+Routing configurations are the building blocks of UDR management. They're used to describe the desired routing behavior for a network group.'''
+metadata owner = 'Azure/module-maintainers'
+
+@sys.description('Conditional. The name of the parent network manager. Required if the template is used in a standalone deployment.')
+param networkManagerName string
+
+@maxLength(64)
+@sys.description('Required. The name of the routing configuration.')
+param name string
+
+@maxLength(500)
+@sys.description('Optional. A description of the routing configuration.')
+param description string?
+
+@sys.description('Optional. A routing configuration contains a set of rule collections that are applied to network groups. Each rule collection contains one or more routing rules.')
+param ruleCollections routingConfigurationRuleCollectionType
+
+resource networkManager 'Microsoft.Network/networkManagers@2024-05-01' existing = {
+  name: networkManagerName
+}
+
+resource routingConfigurations 'Microsoft.Network/networkManagers/routingConfigurations@2024-05-01' = {
+  name: name
+  parent: networkManager
+  properties: {
+    description: description ?? ''
+  }
+}
+
+module routingConfigurations_ruleCollections 'rule-collection/main.bicep' = [
+  for (ruleCollection, index) in ruleCollections ?? []: {
+    name: '${uniqueString(deployment().name)}-RoutingConfigurations-RuleCollections-${index}'
+    params: {
+      networkManagerName: networkManager.name
+      routingConfigurationName: routingConfigurations.name
+      name: ruleCollection.name
+      description: ruleCollection.?description ?? ''
+      appliesTo: ruleCollection.appliesTo
+      disableBgpRoutePropagation: ruleCollection.?disableBgpRoutePropagation ?? true
```

```suggestion
      disableBgpRoutePropagation: ruleCollection.?disableBgpRoutePropagation
```
Redundant as the deployment will default to the parameters default (`true`) regardless if not provided.